frontend/src/lib/utils/eventUsageLogic.ts

```diff
@@ -428,6 +428,9 @@ export const eventUsageLogic = kea<
             loadTime,
             error,
         }),
+        reportInsightOpenedFromRecentInsightList: true,
```

Comment:
can we rename these to something prefixed with `homepage` like `reportHomepageRecentInsightOpened` so that it's more clear/identifiable that these are all instrumentations related to the homepage?

Reply:
Good question, I like to follow the pattern that the event name is describing the action, and the location that the action happened is done with the event properties (via `pathname` + `current_url`). That way if we start using the component elsewhere on the app, it's still traceable.

Reply:
Right but it's a bit unclear unless you go and look at the implementation. Can we do a `reportHomepageInsightOpenedFromRecentInsightList`? This way there's no confusion over what "recent insight list" we're referring to.

Reply:
not really hung up on this btw if you feel strongly about what you have, just thought it'd be more readable :^)

Reply:
Yeah, I definitely see the readability side of things, but I also want to make the event name somewhat future proof. For example, I could see us moving recent insights to the insights page. Or adding it to a user's profile page, and it'd be nice to be able to trace that analytics across all of those pages.